mods/disable_tls.yaml

```diff
@@ -30,3 +30,26 @@ radar_push_endpoint:
   disable_tls: true
 ccSchemaRegistryProxy:
   disable_tls: true
+radar_self_enrolment_ui:
+  disable_tls: true
+radar_kratos:
+  kratos:
+    dev: true
```

Comment:
Just comparing radar_kratos and radar_hydra setup for `dev` property here, either it should be `radar_kratos.kratos.kratos.dev: true` or line 51 should be level higher (`radar_hydra.hydra.dev`), right?

Reply:
The dev property is not mapped to anything. I removed it.